src/pages/Graph/GraphPage.tsx

```diff
@@ -402,6 +368,7 @@ export class GraphPage extends React.Component<GraphPageProps> {
     if (this.loadPromise) {
       this.loadPromise.cancel();
     }
+    const queryTime: TimeInSeconds | undefined = !!this.props.replayQueryTime ? this.props.replayQueryTime : undefined;
```

Comment:
shouldn't this be `undefined` by default?

Reply:
How would you propose to rewrite this keeping it a const?